movie_library/models/actor.py

```diff
@@ -0,0 +1,38 @@
+from django.db import models
+#from .movie import Movie
+
+
+class Actor(models.Model):
+
+    class Meta:
+
+       db_table = "actor"
+
+    class Gender(models.TextChoices):
+        FEMALE = 'Female'
+        MALE = 'Male'
+        OTHER = 'Others'
+
+    first_name = models.CharField(max_length=30)
+    last_name = models.CharField(max_length=30)
+    gender = models.CharField(max_length=6, choices=Gender.choices, default=Gender.FEMALE,)
+    #movies = models.ManyToManyField(Movie)
+
+    '''def show_movies(self):
+        return Actor_Movie.objects.filter(actors=self)'''
+
+    def __str__(self):
+          return f"{self.first_name} {self.last_name}"
+
+
+'''class Actor_Movie(models.Model):
```

Comment:
this approach is better than what you did in movie table 
lets talk about using many to many through